### PR TITLE
Implement string escaping and fix bug with error messages

### LINKED ISSUE #25: No possibility to escape characters in strings
```
s2: String = "no escaping \n \t";
```

Does not compile:

```
Syntax error: expected "\""
```

### LINKED ISSUE #53: Contract does not work if we have ` in throw message
```
    receive() {
        require(1 == 2, "condition can`t be...");
    }
```
After compile message wrap to same symbol like this
```
`target can`t be more than 10`
```
and code died

### PR DESCRIPTION
**Pull request by Gusarich**
This PR resolves #25 and fixes #53 

Supported escaping sequences:
- `\\`
- `\"`
- `\n`
- `\r`
- `\t`
- `\v`
- `\b`
- `\f`
- `\u{ABC}`
- `\uABCD`
- `\xAB`

### CODE REVIEW

File: src/grammar/grammar.ohm [hunk 1]
```diff
@@ -215,8 +215,17 @@ Tact {
     boolLiteral = ("true" | "false") ~idPart
 
     // String literal
-    stringLiteralCharacter = ~("\"" | "\\" | lineTerminator) any
-    stringLiteral = "\"" stringLiteralCharacter* "\""
+    // stringLiteralCharacter = ~("\"" | "\\" | lineTerminator) any
+    // stringLiteral = "\"" stringLiteralCharacter* "\""
```
Comment (anton-trunov) [inline src/grammar/grammar.ohm]:
```suggestion
```

File: src/grammar/grammar.ohm [hunk 2]
```diff
@@ -215,8 +215,17 @@ Tact {
...
+    stringLiteral = "\"" (nonQuoteOrBackslashChar | escapeSequence)* "\""
```
Comment (anton-trunov) [inline src/grammar/grammar.ohm:218]: Let's support all the things from the lexical grammar of JS: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#string_literals.

It has more escape sequences including Unicode, and also more ASCII things, like `\v`, etc. And it allows to split long strings across multiple lines.

**Review by anton-trunov — CHANGES_REQUESTED** [review]
Great start! Let's support some more things though

File: src/grammar/grammar.ohm (hunk 2 above)
Comment (novusnota) [reply src/grammar/grammar.ohm:218]: For an example of this in Ohm, check out [`ohm-grammar.ohm`](https://github.com/novusnota/vscode-ohm/blob/main/samples/ohm-grammar.ohm#L93-107) or [`es5.ohm#L149`](https://github.com/ohmjs/ohm/blob/main/examples/ecmascript/src/es5.ohm#L149)

**Review by novusnota — DISMISSED** [review]

**Review by anton-trunov — APPROVED** [review]
Good stuff 👍

**Review by anton-trunov — CHANGES_REQUESTED** [review]
We need to update CHANGELOG.md

**Pull request merged**